lib/active_merchant/billing/gateways/ebanx.rb

```diff
@@ -48,7 +48,9 @@ def purchase(money, payment, options={})
         add_customer_data(post, payment, options)
         add_payment(post, payment)
         add_address(post, options)
-
+        if post[:payment][:country] == 'BR'
+          add_customer_responsible_person(post, payment, options)
```

Comment:
I think `add_customer_responsible_person(post, payment, options) if post[:payment][:country] == 'BR'` would be slightly preferred despite being a long line.